ci/pipeline-dev.yml

```diff
@@ -53,10 +53,13 @@ test: &test
     config:
       platform: linux
       image_resource:
-        type: docker-image
+        type: registry-image
         source:
-          repository: cloudgov/concourse-dcind
-          tag: original
+          aws_access_key_id: ((ecr-aws-key))
+          aws_secret_access_key: ((ecr-aws-secret))
+          repository: pages-dind-v25
```

Comment:
It looks like we are using the old `entrypoint.sh` in `ci/docker/`.  We'll want to use [this script](https://github.com/cloud-gov/pages-images/blob/main/dind/v25/dockerd-entrypoint.sh) as the entrypoint and it is already baked into the image. We'll need to rework the test task.

Reply:
I'm looking at your work on `pages-proxy`.